lib/widget/input/dropdown.dart

```diff
@@ -61,7 +60,7 @@ abstract class SelectOne extends StatefulWidget
   @override
   Map<String, Function> setters() {
     return {
-      'value': (value) => _controller.maybeValue = value,
+      'value': (value) => _setValue(value),
```

Comment:
if you put this method on the controller, you can force the widgetState to call setState() using `notifyListeners()` inside Controller